crates/subspace-farmer/src/utils/farmer_piece_getter.rs

```diff
@@ -86,6 +92,23 @@ where
             }
         }
 
+        // Try node before reaching to DSN
```

Comment:
We should either move the code or change the comment. The first piece acquisition is accessing the L2 via DSN.